run.py

```diff
@@ -13,21 +14,32 @@ def main():
     logger = logging.getLogger(__name__)
     logger.setLevel(logging.DEBUG)
 
-    # Start the server
     if not get_server_addr_file().is_file():
-        logger.debug('Starting Server...')
-        Popen('python server.py', shell=True)
+        logger.debug("Starting server...")
+        if os.name == "nt":
+            subprocess.Popen(
+                "pythonw server.py",
```

Comment:
Why pythonw?

Reply:
If we use `python`, it will open a command prompt window. Closing it will terminate the process.